### PR TITLE
[Port] Add Product Feedback survey link to welcome page (#14705)

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Closes https://github.com/microsoft/azuredatastudio/issues/14738

Links to https://aka.ms/AzureSQLSurvey

image

### CODE REVIEW

**Comment by Charles-Gagnon** [comment]
This was approved by Ken so once it passes checks I'm planning on merging it in

**Review by kburtram — APPROVED** [review]

**Review by chlafreniere — APPROVED** [review]

**Pull request merged**
